Sources/Components/Expandable/Expandable.swift

```diff
@@ -0,0 +1,142 @@
+import SwiftUI
+
+extension Warp {
+    /**
+
+    Expandable allows users to interact with expandable/collapsible sections of content. Also known as “accordion”
+
+    **When to use**
+
+     Use the Expandable component when presenting content that can be optionally revealed to users.
+     Consider it for long content that may overwhelm users if displayed all at once.
+
+     **Usage**
+
+     Provide the `Warp.Expandable` with:
+
+     - A title.
+     - A subtitle.
+     - A `Warp.ExpandableStyle`, choose from:
+        - `default`: for a non bordered expandable.
+        - `box`: for a bordered expandable with rounded corners.
+        - `boxBleed`: for a bordered expandable with sharp corners.
+     - A boolean `isAnimated` to determine if the expand/collapse transition is animated or not. **Optional** _default value is `true` if none is specified_.
+     - A `ColorProvider`. **Optional:** _default is read from `Config.colorProvider` if none is specified_.
+     */
+    public struct Expandable: View {
+        /**
+        **Collapsed by default**
+
+         Expendables begins in the collapsed state with all content panels closed.
+         Starting in a collapsed state gives the user a high-level overview of the available information.
+        */
+        @State private var isExpanded: Bool = false
+
+        let title: String
+        let subtitle: String
```

Comment:
On other platforms, it's possible to pass a view to the Expendable component. Should we do the same, or is a string of text enough ?

Reply:
I can make it so we take a `ViewBuilder` as parameter...I think I'll make both available then, so that you can pass both a `String` and an entire View. Let me give that a go.

Reply:
Looks great 👍